Source/ACE/Entity/Container.cs

```diff
@@ -26,48 +29,111 @@ public Container(ObjectType type, ObjectGuid guid, string name, ushort weenieCla
         public Container(AceObject baseObject)
             : base(baseObject) { }
 
+        public Container(ObjectGuid guid, AceObject baseAceObject)
```

Comment:
I'm a bit puzzled why this constructor is the best solution.  The vast majority of these properties sets are setting properties on a default AceObject of the base class.  It seems that there would have been better solutions than doing explicit copies of everything (perhaps calling the other base constructor with a clone of the baseAceObject, then overriding the Guid?)